### PR TITLE
Fixes #14256 - Update the strings to be more compliant with the proje…

### PR DESCRIPTION
**Pull request by bkearney**
…ct standards

### CODE REVIEW

File: app/models/concerns/foreman_openscap/openscap_proxy_core_extensions.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def update_scap_client_params(proxy_id)
       new_proxy = SmartProxy.find proxy_id
       model_match = self.class.name.underscore.match(/\Ahostgroup\z/) ? "hostgroup" : "fqdn"
       puppetclass = Puppetclass.find_by_name("foreman_scap_client")
-      fail _("Puppetclass 'foreman_scap_client' not found, make sure it is imported form Puppetmaster") if puppetclass.nil?
+      fail _("Puppet class 'foreman_scap_client' not found, make sure it is imported form Puppetmaster") if puppetclass.nil?
```
Comment (domcleal) [inline app/models/concerns/foreman_openscap/openscap_proxy_core_extensions.rb]: "Puppet master" too

**Comment by domcleal** [comment]
<pre>
#: ../app/helpers/policy_dashboard_helper.rb:26
msgid "Incompliant hosts"
</pre>

Should that be non-compliant?

<pre>
#: ../app/models/foreman_openscap/compliance_status.rb:22
#: ../app/views/foreman_openscap/policy_mailer/_dashboard.erb:17
msgid "Incompliant"
</pre>

ditto?

<pre>
#: ../app/models/foreman_openscap/scap_content.rb:9
msgid "No Proxy with OpenSCAP features"
#: ../app/models/foreman_openscap/scap_content.rb:14
</pre>

Bit of inconsistent capitalisation.

<pre>
#: ../app/views/arf_reports/delete_multiple.html.erb:28
msgid "these Complinace reports"
</pre>

Spelling.

<pre>
#: ../app/views/arf_reports/show.html.erb:4
msgid "Reported at %s "
</pre>

Trailing whitespace.

<pre>
#: ../app/views/compliance_hosts/show.html.erb:3
msgid "%s compliance reports by policy"
</pre>

Missing plural forms, use n_().

<pre>
#: ../app/views/dashboard/_compliance_host_reports_widget.html.erb:12
msgid "Othered|O"
msgstr ""
</pre>

Should that say "Others" or something?  These should have a translator comment like Foreman core, e.g. `# TRANSLATORS: initial character of Other`.

<pre>
#: ../app/views/policies/welcome.html.erb:11
msgid ""
"In Foreman, a compliance policy checklist is defined via %s, once SCAP content"
" is present, you can create a policy,\n"
"            assign select hostgroups and schedule to run."
msgstr ""
</pre>

"hostgroups" should be "host groups", and like the other in this file (and app/views/scap_contents/welcome.html.erb), it might be best to remove the whitespace from the source string.  I'm unsure what %s here is too.

**Comment by bkearney** [comment]
@domcleal Made most of the changes. Some I did not:

* Incompliant - This seems to be an SCAP word, and it is valid per Websters.
* I could not get the comments from the html to make it into the translator. 

The rest are in, and I have force pushed the changes.

**Comment by shlomizadok** [comment]
@bkearney - thanks for this.
The tests are failing because of texts changes. OpenScap should be OpenSCAP in the tests (I can change this for you if you want).

@domcleal do you ack the changes?

**Comment by domcleal** [comment]
> msgid "No Proxy with OpenSCAP features"
> Bit of inconsistent capitalisation.

Looks unfixed, lower case "proxy".

> msgid "Othered|O"
> Should that say "Others" or something?

Unfixed?  Or is that more app-specific terminology?

> I could not get the comments from the html to make it into the translator.

Foreman puts them into a helper, which extracts properly: https://github.com/theforeman/foreman/blob/1.11.0-RC2/app/helpers/dashboard_helper.rb#L95

**Comment by shlomizadok** [comment]
"Othered" is a terminology used by OpenSCAP. We can change it though to a proper English word.

**Comment by domcleal** [comment]
It's just if you don't change it, adding a comment to explain what it means & how it should be translated would assist translators.

**Comment by bkearney** [comment]
I left Othered there for that reason. I dont know OpenSCAP well enough to to suggest changing it.

**Comment by bkearney** [comment]
@domcleal that strings in in html, so I tried to add the color but it seems to have not taken it.

**Comment by domcleal** [comment]
If it's in a view it can be put straight into a helper, then should extract fine.

**Comment by shlomizadok** [comment]
@bkearney - Could you kindly change also the texts in [test/unit/scap_content_test.rb (line 18)](https://github.com/theforeman/foreman_openscap/blob/master/test/unit/scap_content_test.rb#L18) so it will fit the new texts? (and make tests pass...)
Thanks

**Comment by shlomizadok** [comment]
@bkearney - would love to merge this once comments (and test) are fixed

**Comment by bkearney** [comment]
@shlomizadok in.. sorry for the delay.

**Comment by bkearney** [comment]
@shlomizadok bump

**Comment by shlomizadok** [comment]
Thank you so much @bkearney !
Thank you @domcleal for reviewing and commenting.

**Pull request merged**